lib/controller.js

```diff
@@ -52,9 +53,25 @@ controller.postActions = (request, response) => {
             }
             response.json(responseJson);
         })
-        .catch(({code, message}) => {
-            response.status(code).json({message: message});
+        .catch((error) => {
+            response.status(error.statusCode).json({message: error.message});
         });
 };
 
+controller.getQrCode = (request, response) => {
```

Comment:
These new functions don't look like other parts of the application. It has magic numbers, missing `;` and different variable naming.

Reply:
Corrected the function name to getQrcode, variable names and placed required ;

Regarding the magic number, I"m trying to get directory path from process working directory value returned from __dirname. For which, I'm extracting the substring from index-0 to the last 5th character hence **substr(0,process_directory.length-4)**